src/feature/guide/GuideItem.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react"
+import styled from "@emotion/styled"
+import { navigate } from "gatsby"
+import { GatsbyImage, StaticImage } from "gatsby-plugin-image"
+import { IGatsbyImageData } from "gatsby-plugin-image/dist/src/components/gatsby-image.browser"
+import Typography from "@mui/material/Typography"
+import Box from "@mui/material/Box"
+import Stack from "@mui/material/Stack"
+import { BorderBox } from "../../components/layout"
+
+type ItemImageProps = {
+  image: IGatsbyImageData
+  alt: string
+}
+
+export type Props = {
+  title: string
+  description: string
+  timeToRead: number
+  image: IGatsbyImageData
+  alt: string
+  courseSlug: string | null
+  slug: string
+}
+
+const StyledBorderBox = styled(BorderBox)`
+  width: 369px;
+  height: 104px;
+  border-radius: 8px;
+  &:hover {
+    background-color: #fafafa;
+    cursor: pointer;
+    filter: drop-shadow(0px 1px 4px rgba(0, 0, 0, 0.25));
+  }
+  @media screen and (min-width: 1024px) {
+    width: 864px;
+    height: 140px;
+  }
+`
+
+export const ItemImage = ({ image, alt }: ItemImageProps) => (
+  <Box
+    width={[`73px`, `247px`]}
+    height={[`41px`, `140px`]}
+    borderRadius={[`4px`, `8px 0 0 8px`]}
+    overflow="hidden"
+    transform="translateZ(0)"
+  >
+    <GatsbyImage style={{ height: `100%` }} image={image} alt={alt} />
+  </Box>
+)
+
+export const GuideItem = ({
+  title,
+  description,
+  timeToRead,
+  image,
+  alt,
+  courseSlug,
+  slug,
+}: Props) => {
+  const handleNavigate = React.useCallback(() => {
+    navigate(`/progetti/${courseSlug}/${slug}`)
+  }, [navigate])
+
+  return (
+    <StyledBorderBox onClick={handleNavigate}>
```

Comment:
Qui bisogna probabilmente wrapparlo in un link in modo che google possa seguire i link e mappare le sezioni. 
Andrei anche usare la props `component='article'` sulle singole cards.